libcudacxx/include/cuda/__ptx/instructions/shfl.h

```diff
@@ -0,0 +1,136 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#if !defined(_CUDA_PTX_SHFL_SYNC_H)
+#  define _CUDA_PTX_SHFL_SYNC_H
```

Comment:
It seems that clang-format does not recognize the header guard in this case
```suggestion
#ifndef _CUDA_PTX_SHFL_SYNC_H
#define _CUDA_PTX_SHFL_SYNC_H
```